File: mobile/src/main/java/com/nomi/artwatch/di/module/Modules.java
package com.nomi.artwatch.di.module;

import com.nomi.artwatch.Application;

/**
 * Created by Ryota Niinomi on 2015/11/03.
 */
public class Modules {
    public static ApplicationModule applicationModule(Application application) {
        return new ApplicationModule(application);
    }
}

File: mobile/src/main/java/com/nomi/artwatch/di/module/Modules.kt
package com.nomi.artwatch.di.module

import com.nomi.artwatch.App

/**
 * Created by Ryota Niinomi on 2015/11/03.
 */
object Modules {
    fun applicationModule(app: App): ApplicationModule {
        return ApplicationModule(app)
    }
}
